chinese/wg-value/focus-areas/organizational-value/organizational-influence.md

```diff
@@ -0,0 +1,50 @@
+###  这个指标是候选基线指标。评审这个指标请看[问题 #146](https://github.com/chaoss/wg-value/issues/146)。根据评论周期，这个指标将在下一个常规版本发布。
```

Comment:
we could remove this line, no translations needed.

Reply:
Done